app/javascript/app.vue

```diff
@@ -8,15 +8,20 @@
       <flash-message transition-name="list-complete" />
     </div>
     <div class="app">
-      <div
-        class="product-list"
-        key="products"
-      >
-        <product
-          v-for="product in sortedProductList"
-          :key="product.id"
-          :product="product"
-        />
+      <div class="product-categories">
+        <div
+          v-for="category in productsArray"
+          :key="category.title"
+        >
+          <span class="product-category__title">{{category.title}}</span>
```

Comment:
Expected 1 space after '{{', but not found   vue/mustache-interpolation-spacing<br>Expected 1 space before '}}', but not found  vue/mustache-interpolation-spacing